### ISSUE TITLE
staking_active: false even when unlocked for staking

### ISSUE THREAD

**Issue by veilgets** [Veil-Project/veil#182]
on windows 7, report from a user:  

I installed latest version, and didn't encrypt the wallet.. it worked fine to stake.. then I encrypted wallet and unlocked, and it always says false now

**Comment by presstab** [comment]
What always says false?

**Comment by veilgets** [comment]
staking_active: false won’t turn to staking_active: true even when unlocking for some

**Comment by cryptaiz** [comment]
Hi. Running on win7 x64. 
I updated my wallet to 1.0.0.5. 
Minted zerocoins. Checked the status for maturity which now says matured. 
The wallet is encrypted yet i have unlocked it. 
The wallet UI shows unlocked for staking (the blue indicator) but when i type getwalletinfo the staking_active is false. 

restarted the wallet. Rescynced the blockchain. Nothing changes. Same issue even before encrypting the wallet with a password.
